lambda/road-link-change-handler/aws/cloudformation/lambda-resources.yaml

```diff
@@ -194,7 +194,7 @@ Resources:
     Properties:
       Name: !Sub "${Environment}-${ApplicationName}-start-road-link-change-handler-event"
       Description: "Scheduled event to start fetching of road link changes"
-      ScheduleExpression: "cron(0 1 ? * 3-6 *)" # Run Tuesday, Wednesday, Thursday, and Friday
+      ScheduleExpression: "cron(30 18 ? * 2-5 *)" # Run Tuesday, Wednesday, Thursday, and Friday
```

Comment:
Kommentin vois korjata. Kesäajalla ei vissiin merkitystä tässä asiassa?

Reply:
Ei sillä mun mielestä pitäis olla merkitystä